server/src/storage/data_point.rs

```diff
@@ -0,0 +1,120 @@
+/* Copyright 2022 The MiniModelarDB Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! Support for a single internal data point. Note that this struct is mainly
+//! used when transitioning from a raw message into the in-memory representation.
+
+use crate::storage::{MetaData, Timestamp, Value};
```

Comment:
I have changed the imports to be grouped using the convention that std is first, then external crates and then internal crates. Since this is not standard in rustfmt, should we add it to the style guide?

Reply:
Looks like we can automate it by setting [group_imports](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#group_imports) to `StdExternalCrate` in rustfmt.toml.